composer.json

```diff
@@ -23,6 +23,7 @@
         "knplabs/gaufrette": "^0.1.6 || ^0.2 || ^0.3",
         "kriswallsmith/buzz": "^0.15",
         "psr/log": "^1.0",
+        "sensio/generator-bundle": "^2.3 || ^3.0",
```

Comment:
Can you make that ^3.1

(We are droping support for SF 2.3 to 2.7) Look: https://github.com/sonata-project/SonataMediaBundle/pull/1307